### ISSUE TITLE
Fix Hyper-Parameters For AutoKeras Standard Method Search Spaces

### ISSUE THREAD

**Issue by WilliamJudge94** [keras-team/autokeras#1480]
### Feature Description
A way to fix certain aspects of the search space such as learning rate, optimizer, etc for every model created by the ImageClassifier.

### Reason
Allows a User to minimize their search space for the standard AutoKeras search methods.

**Comment by haifeng-jin** [comment]
This feature will be enabled by #1494.
